pkg/reconciler/builder/builder.go

```diff
@@ -173,6 +181,17 @@ func (c *Reconciler) reconcileBuilder(ctx context.Context, builder *buildapi.Bui
 		},
 	}, builder.NamespacedName())
 
+	c.Tracker.Track(reconciler.Key{
+		NamespacedName: types.NamespacedName{
+			Name:      builder.Spec.Lifecycle.Name, // TODO: confirm this is what we want
```

Comment:
Right? Is there a better name for "Name" ...like maybe "ImageName" or "ImageTag"?